feat: qrcode support render svg (#42570)
ant-design/ant-design: components/qrcode/demo/type.md

## zh-CN

> 通过设置 `type` 自定义渲染类型。二维码上鼠标右键->检查->源码中查看渲染结果。

Reviewer: ```suggestion
通过设置 `type` 自定义渲染结果。提供 `canvas` 和 `svg` 两个选项。
```

> 通过设置 `type` 自定义渲染结果。提供 `canvas` 和 `svg` 两个选项。

Reviewer: ```suggestion
通过设置 `type` 自定义渲染结果，提供 `canvas` 和 `svg` 两个选项。
```

Author: 这都是AI找出来的吗？太厉害了！！

## en-US

> Render Type.

Reviewer: 英文描述需要写详细一点

> Customize the rendering results by setting 'type'. Provide two options: 'canvas' and' svg '.

Reviewer: ```suggestion
Customize the rendering results by 'type', provide options 'canvas' and 'svg'.
```
